pkg/signatures/types.go

```diff
@@ -1,7 +1,49 @@
 package signatures
 
-// SignatureIntegrationIDPrefix should be prepended to every human-hostile ID of a
-// signature integration for readability, e.g.,
-//
-//	"io.stackrox.signatureintegration.94ac7bfe-f9b2-402e-b4f2-bfda480e1a13".
-const SignatureIntegrationIDPrefix = "io.stackrox.signatureintegration."
+import (
+	"github.com/stackrox/rox/generated/storage"
+)
+
+const (
+	// SignatureIntegrationIDPrefix should be prepended to every human-hostile ID of a
+	// signature integration for readability, e.g.,
+	//
+	//	"io.stackrox.signatureintegration.94ac7bfe-f9b2-402e-b4f2-bfda480e1a13".
+	SignatureIntegrationIDPrefix = "io.stackrox.signatureintegration."
+
+	releaseKey3PublicKey = `-----BEGIN PUBLIC KEY-----
```

Comment:
If you have a Jira related to fetching this dynamically, can you add it here as a TODO?